docs/helm-chart-checks.md

```diff
@@ -291,11 +291,14 @@ A profile defines a set of checks to run and an indication of whether each check
 - community
   - Defines the requirements for a community chart to pass helm chart certfication.
   - The ```helm-lint``` check is the only mandatory check with all other checks optional.
+- developer-console
+  - Defines the requirements for a developer-console chart to pass helm chart certfication.
+  - The ```contains-test``` check is an optional check . The ```chart-testing``` and ```images-are-certified``` are checks which are not enabled for this profile.
```

Comment:
You have also removed required-annotations-present

Reply:
Yeah i did, because we had removed all annotations from the profile yaml so it did not make sense to me, so after discussion with David removed that. Let me know if we need to revert.